app/upgrades/v1_0_3/constants.go

```diff
@@ -0,0 +1,6 @@
+package v1_0_3
+
+const (
+	UpgradeName = "v1.0.3"
+	UpgradeInfo = "{\\\"binaries\\\":{\\\"linux/amd64\\\":\\\"http://swisstronik-server/swisstronikd_v1.0.3.tar.gz?checksum=sha256:05c3372cbe5fb54b53de5075b7f92ca174e44cc95ab0dec0e1481cbd4339b46a\\\"}}"
```

Comment:
Can we put empty `UpgradeInfo`, since it is not possible to automatically download binaries

Reply:
Okay, it is not used anymore, just wanted to keep it as an example.